code/modules/jobs/job_types/adventurer/types/antag/brigand.dm

```diff
@@ -34,9 +34,10 @@
 	neck = /obj/item/clothing/neck/coif
 	head = /obj/item/clothing/head/helmet/leather/volfhelm
 	armor = /obj/item/clothing/armor/leather/hide
+	H.mind.special_role = "Bandit"
 	H.change_stat(STATKEY_STR, 3)
-	H.change_stat(STATKEY_END, 2)
-	H.change_stat(STATKEY_CON, 2)
+	H.change_stat(STATKEY_END, 3)
+	H.change_stat(STATKEY_CON, 3)
```

Comment:
So a malnourished, poorly fed vagabond got health of a royal knights or a templar?